### ISSUE TITLE
Update documentation to dotnet interactive to 4.2.1

### ISSUE THREAD

**Issue by EverybodyKurts** [plotly/Plotly.NET#445]
## Description

Using version 4.2.0 in a polyglot or jupyter notebook will result in an error being thrown. Using version 4.2.1 fixes this.

This version bump should be reflected in the documentation.

**Comment by kMutagene** [comment]
Good catch, i think the main issue here is that we can only automatically include one version in our docs release process, but the versions of the individual packages have diverged quite a lot. Especially keeping the interactive extension working required lots of additional patches in the past because polyglot notebook API changed so often. I think we should just unlist v4.2.0 of the interactive extension and do not use version numbers in the install instructions at all, since using latest here is always the best approach anyways
